apps/api/src/sandbox/managers/sandbox.manager.ts

```diff
@@ -929,20 +929,54 @@ export class SandboxManager {
         return SYNC_AGAIN
       }
 
-      //  exclude the runner that the last runner sandbox was on
-      const availableRunners = (
-        await this.runnerService.findAvailableRunners({
+      //  make sure we pick a runner that has the base snapshot
+      const baseSnapshot = sandbox.snapshot
+        ? await this.snapshotService.getSnapshotByName(sandbox.snapshot, sandbox.organizationId)
```

Comment:
If an organization deleted a snapshot after creating and archiving a sandbox, this would return an error. We should try-catch it